packages/server/src/api/todo/adx/account/index.ts

```diff
@@ -1,11 +1,14 @@
-import { Server } from '../../../lexicon'
+import { Server } from '../../../../lexicon'
 import { InvalidRequestError } from '@adxp/xrpc-server'
-import * as locals from '../../../locals'
+import * as locals from '../../../../locals'
 import { Repo } from '@adxp/repo'
 import { PlcClient } from '@adxp/plc'
-import { InviteCode, InviteCodeUse } from '../../../db/invite-codes'
+import { InviteCode, InviteCodeUse } from '../../../../db/invite-codes'
+import passwordReset from './password-reset'
 
 export default function (server: Server) {
+  passwordReset(server)
```

Comment:
nit, but im inclined to either
- not nest these directories & just have `password-reset` at the same level as `account` (similar to `invites`)
- or more aggressively nest directories & have a separate file for every xrpc method

leaning towards the former